yarn.lock

```diff
@@ -8980,11 +8985,16 @@ lodash.xorby@^4.7.0:
   resolved "https://registry.yarnpkg.com/lodash.xorby/-/lodash.xorby-4.7.0.tgz#9c19a6f9f063a6eb53dd03c1b6871799801463d7"
   integrity sha1-nBmm+fBjputT3QPBtocXmYAUY9c=
 
-lodash@4.17.20, "lodash@>=3.5 <5", lodash@^4.17.11, lodash@^4.17.13, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.19, lodash@^4.17.20, lodash@^4.17.5, lodash@~4.17.10:
+"lodash@>=3.5 <5", lodash@^4.17.11, lodash@^4.17.13, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.19, lodash@^4.17.20, lodash@^4.17.5, lodash@~4.17.10:
   version "4.17.20"
   resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.20.tgz#b44a9b6297bcb698f1c51a3545a2b3b368d59c52"
   integrity sha512-PlhdFcillOINfeV7Ni6oF1TAEayyZBoZ8bcshTHqOYJYlrqzRK5hagpagky5o4HfCzzd1TRkXPMFq6cKk9rGmA==
 
+lodash@^4.17.21:
+  version "4.17.21"
+  resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.21.tgz#679591c564c3bffaae8454cf0b3df370c3d6911c"
+  integrity sha512-v2kDEe57lecTulaDIuNTPy3Ry4gLGJ6Z1O3vE1krgXZNrsQ+LFTGHVxVjcXPs17LhbZVGedAJv8XZ1tvj5FvSg==
+
```

Comment:
The version 4.17.21 is added in addition to version 4.17.20 so there are now two versions of `lodash`. But to me it seems that version 4.17.21 satisfies all the version requirements (lodash@>=3.5 <5", lodash@^4.17.11, lodash@^4.17.13, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.19, lodash@^4.17.20, lodash@^4.17.5, lodash@~4.17.10, lodash@^4.17.21). So the version 4.17.20 isn't needed anymore? Can it be removed from `yarn.lock`?

Reply:
Yarn lock should not be edited manually. This seems to be an issue with yarn. I re-installed packages by removing lock file and node_modules and this was fixed. But it changed a lot of other dependencies too that are not related to this PR. 

So I did the same with "develop"-branch and created another PR: https://github.com/City-of-Helsinki/open-city-profile-ui/pull/179. I'll rebase this branch to that and run "yarn install" again. That will probably fix this.

Reply:
Ran yarn with "clean" install and looks better now

Reply:
> Yarn lock should not be edited manually.

Yep, I meant that "force yarn to do the correct thing somehow". That's what you did. Well done :+1: